apps/docs/src/app/core/component-docs/table/examples/table-navigatable-row-example.component.html

```diff
@@ -0,0 +1,103 @@
+            <td fd-table-cell>{{ row.column2 }}</td>
+            <td fd-table-cell>{{ row.column3 }}</td>
+            <td fd-table-cell>{{ row.date }}</td>
+            <td fd-table-cell [fitContent]="true" [noPadding]="true">
+                <i fd-table-icon [glyph]="getGlyph(row)" [navigation]="row.navigatable" role="presentation"></i>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+<br>
+
+<h4 class="table-example-popin-title">Row with navigation button</h4>
+<table fd-table>
+    <thead fd-table-header>
+        <tr fd-table-row>
+            <th fd-table-cell>Column 1</th>
+            <th fd-table-cell>Column 2</th>
+            <th fd-table-cell>Column 3</th>
+            <th fd-table-cell>Column 4</th>
+            <th fd-table-cell></th>
+        </tr>
+    </thead>
+    <tbody fd-table-body>
+        <tr *ngFor="let row of tableRowsAdditional" fd-table-row [hoverable]="true" [navigatable]="row.navigatable">
+            <td fd-table-cell>
+                <a fd-link href="#">{{ row.column1 }}</a>
+            </td>
+            <td fd-table-cell>
+                <p fd-table-text>
+                    {{ row.column2 }}
+                </p>
+            </td>
+            <td fd-table-cell>{{ row.date }}</td>
+            <td fd-table-cell><fd-icon [glyph]="row.type"></fd-icon></td>
+            <td fd-table-cell [fitContent]="true">
+                <button aria-label="navigation" class="fd-button fd-button--transparent" [disabled]="!row.navigatable" [routerLink]="'/core/home'">
+                    <i [ngClass]="getClass()"></i>
+                </button>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+<div style="max-width: 450px; padding-bottom: 50px">
+    <h4 class="table-example-popin-title">Mobile Mode</h4>
+    <table fd-table [popIn]="true" [noBorderX]="true" [noBorderY]="true" [responsive]="true">
+        <thead fd-table-header>
+        <tr fd-table-row>
+            <th fd-table-cell>Fruit Description</th>
+            <th fd-table-cell>Price</th>
+            <th fd-table-cell></th>
+        </tr>
+        </thead>
+        <tbody fd-table-body>
+        <ng-container *ngFor="let fruit of fruits">
+            <tr fd-table-row [main]="true" [activable]="true" [navigatable]="fruit.navigatable" (click)="alert(fruit)"> 
+                <td fd-table-cell>
+                    <p fd-table-text [title]="true">{{fruit.name}}</p>
+                    <p fd-table-text>{{fruit.country}}</p>
+                </td>
+                <td fd-table-cell [fitContent]="true">
+                    {{fruit.price}}
+                </td>
+                <td fd-table-cell [fitContent]="true" [noPadding]="true">
+                    <i fd-table-icon [glyph]="getGlyph(fruit)" [navigation]="fruit.navigatable" role="presentation"></i>
+                </td>
+            </tr>
+            <tr fd-table-row [secondary]="true" [unnavigable]="fruit.unnavigable">
```

Comment:
`unnavigable` is not a valid input (leftover from previous commit?)

Reply:
Thanks Mike. Fixed it.